interface/wx/dcsvg.h

```diff
@@ -133,15 +95,20 @@ class wxSVGFileDC : public wxDC
 
     //@{
     /**
-        Functions not implemented in this DC class.
+        Function not implemented in this DC class.
```

Comment:
Is this intentional? The plural seems more appropriate to me, but perhaps I'm missing something here.

Reply:
This is intentional. The text is used for each function separately in the generated docs, see https://docs.wxwidgets.org/trunk/classwx_s_v_g_file_d_c.html#afc23fca3c1919a917ba4fa4ea1a47bd6